ts/handlers/html/HTMLDomStrings.ts

```diff
@@ -241,6 +241,18 @@ export class HTMLDomStrings<N, T, D> {
     return [next, ignore];
   }
 
+  /**
+   * Handle an unknown node type (nodeType other than 1, 3, 7)
+   *
+   * @param {N} node           The node to process
+   * @param {boolean} ignore   Whether we are currently ignoring content
+   * @return {N|T}             The next element to process
+   */
+  protected handleOther(node: N, _ignore: boolean): N | T {
```

Comment:
Why do we need `_ignore`?

Reply:
This is for in case someone needs to subclass this class in order to handle additional node types, so they have access to whether the node is being ignored or not.